integration/experiment/gpu_frequency_sweep/gen_gpu_activity_constconfig_recommendation.py

```diff
@@ -83,8 +96,19 @@ def get_config_from_frequency_sweep(full_df):
     parser = argparse.ArgumentParser()
     parser.add_argument('--const-config-path', required=False, default=None,
                         help='path containing existing ConstConfigIO configuration file')
+    parser.add_argument('--gpu-energy-margin', default=0, type=float, dest='gpu_energy_margin',
+                        help='Percentage of additional energy it is acceptable to consume if it results '
+                             'in a lower frequency selection for Fe.  This is useful for analyzing '
+                             'noisy systems that have many gpu frequencies near the Fe energy consumption value')
     parser.add_argument('--path', required=True,
                         help='path containing reports and machine.json')
+    parser.add_argument('--use-requested-frequency', action='store_true', default=False,
+                        dest='use_freq_req',
+                        help='Use the frequency that was requested during the frequency sweep instead '
+                             'of the achieved frequency for a given run.  This is useful in cases where '
+                             'multiple frequency domains or settings are impacted (i.e. core frequency causes '
+                             'an uncore frequency change) and the achieved frequency does not reflect this '
+                             'behavior.')
```

Comment:
```suggestion
                             'behavior.'
```

Reply:
I'm not sure I understand this one, the ) here is the ending parentheses of the parser add_argument.